timeseries/src/main/scala/com/criteo/cuttle/timeseries/CuttleProject.scala

```diff
@@ -86,6 +90,10 @@ class CuttleProject private[cuttle] (val name: String,
       if (paused) {
         logger.info("Pausing workflow")
         scheduler.pauseJobs(jobs.all, executor, xa)(Auth.User("Startup"))
+      } else {
+        val toPause = jobs.all.filter(_ == jobsToBePausedOnStartup)
```

Comment:
Unless mistaking you are comparing a `Set[Job]` with `Job` (`_ == jobsToBePausedOnStartup`), maybe `jobs.intersect(jobsToBePausedOnStartup)` or something like?
Thanks for your contribution :)

Reply:
you are very right 🤦‍♂